packages/website/src/bin/page-generator/test.js

```diff
@@ -60,20 +74,20 @@ describe('Generate pages', () => {
   describe('package pages generation', () => {
     it('creates a home page per package', () => {
       expect(
-        fs.existsSync(
+        getOutput(
           path.join(pagesPath, 'packages', 'mock-package1', 'index.js'),
         ),
-      ).toEqual(true);
+      ).toMatchSnapshot();
```

Comment:
In general I'm loathe to add `toMatchSnapshot` - I would much rather use `toMatchInlineSnapshot`, as it is less prone to "I expected things to change so I updated all snapshots instead of looking at them" which I have seen with just `toMatchSnapshot`

Reply:
I think `toMatchInlineSnapshot` is good for cases where your snapshot is pretty small in size but these page templates are a bit borderline imo and  somewhat noisy.

I think where inline snapshots shine are co-location of test + output so its easier to review changes and reduce context switching. Imo, devs are just as likely to update all snapshots without looking at them whether they are inline or not, the jest CLI output is exactly the same. I also think devs and reviewers should be actively reviewing these changes and [treating them like code](https://jestjs.io/docs/en/snapshot-testing#1-treat-snapshots-as-code).

One concrete blocker from adopting inline snapshots in this test file is the `assertDocs` helper functions that we use in certain places, you can't place inline snapshot assertions in helper functions.

I'd rather use snapshots for everything in the file for consistency's sake since some tests won't be able to use them and due to the size/noise. What do you think?